safety_distance: 0.08

rgb_topic: /hsrb/head_rgbd_sensor/rgb/image_rect_color
depth_topic: /hsrb/head_rgbd_sensor/depth_registered/image_rect_raw # 16UC1 with depth in mm
cam_info_topic: /hsrb/head_rgbd_sensor/depth_registered/camera_info
point_cloud_topic: '/hsrb/head_rgbd_sensor/depth_registered/rectified_points'
global_frame: '/map'

detector_pose_estimator:
  class_confidence_threshold: 0.3
  unknown_object_grasp_detector_topic: /pose_estimator/find_grasppose_haf
  detector_topic: /get_objects_on_table/get_label_image #'/pose_estimator/find_grasppose'
  timeout_duration: 40.0